pkgs/by-name/kt/kty/package.nix

```
@@ -0,0 +1,42 @@
+{
+  rustPlatform,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kty";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "grampelberg";
+    repo = "kty";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-E9PqWDBKYJFYOUNyjiK+AM2WULMiwupFWTOQlBH+6d4=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  env = {
+    OPENSSL_NO_VENDOR = 1;
+  };
+
+  buildInputs = [
+    openssl
+  ];
```

```suggestion
  buildInputs = [
    openssl
  ] ++ lib.optionals stdenv.isDarwin [
    darwin.apple_sdk.frameworks.SystemConfiguration
  ];
```

`lib.optionals (stdenv.isDarwin)` has redundant parenthesis